### PR TITLE
desktop: use dropdowns for group setting message duration instead of wheel picker

### PR DESCRIPTION
**Pull request by jessehorne**
duration

### CODE REVIEW

**Comment by jessehorne** [comment]
I think the text should be bigger for the dropdowns.

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/helpers/CustomTimePicker.kt [hunk 1]
```diff
@@ -183,18 +174,41 @@ fun CustomTimePickerDialog(
...
-            timeUnitsLimits
-          )
 
-          TextButton(onClick = { confirmButtonAction(selection.value) }) {
-            Text(
-              confirmButtonText,
-              fontSize = 18.sp,
-              color = MaterialTheme.colors.primary
-            )
+          fun getUnitValues(unit: CustomTimeUnit, selectedValue: Int): List<Int> {
```
Comment (epoberezkin) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/helpers/CustomTimePicker.kt:178]: please move it to avoid diff

**Comment by epoberezkin** [comment]
let's please move it within the boundaries of app window and avoid rectangular white background that sticks behind rounded corners

**Comment by epoberezkin** [comment]
... so it looks like the first menu:

image

**Pull request closed without merging**